src/olympia/addons/tests/test_update.py

```diff
@@ -91,12 +91,14 @@ def test_no_version(self):
         assert up.is_valid()
 
     def test_unlisted_addon(self):
-        """Don't provide updates for unlisted addons."""
+        """Add-ons with only unlisted versions are valid, they just don't
```

Comment:
It will be valid, as long as it has not been deleted or disabled by the author(s).